app/views/refinery/acts_as_indexed/admin/_search.html.erb

```diff
@@ -1,5 +1,10 @@
 <form method='GET' action='<%= url %>' class='search_form'>
-  <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => params[:search], :title => t('search_input_notice', :scope => 'refinery.admin.search') %>
+  <%= text_field :search, nil, :id => 'search', :type => 'search',
+    :name => 'search',
+    :value => params[:search],
+    :title =>       t('search_input_help', :scope => 'refinery.admin.search', :query => h(params[:search])).html_safe,
```

Comment:
The new key is in https://github.com/refinery/refinerycms/issues/2960. However it is only in 'en.yml'. 
Should  I add it to the other locales, and how? (i.e. in English, learn some new languages, with a special 'needs translation' marker?)

I think I asked the question somewhere else about how we get new keys added to different locales.